atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbKvs.java

```diff
@@ -564,6 +566,22 @@ public void checkAndSet(CheckAndSetRequest checkAndSetRequest) throws CheckAndSe
         }
     }
 
+    @Override
+    public void multiCheckAndSet(MultiCheckAndSetRequest multiCheckAndSetRequest) throws MultiCheckAndSetException {
+        TableReference tableReference = multiCheckAndSetRequest.tableRef();
+        Map<Cell, byte[]> oldValueMap = multiCheckAndSetRequest.expected();
+
+        multiCheckAndSetRequest.updates().forEach((cell, newVal) -> {
```

Comment:
We need to implement this properly, otherwise DBKVS might explode when we try to use txn4. It is ok to leave it as a todo for now, but we should track it